src/sentry/lang/native/sources.py

```diff
@@ -449,12 +446,7 @@ def get_sources_for_project(project):
     if sources_config:
         try:
             custom_sources = parse_sources(sources_config)
-            sources.extend(
-                normalize_user_source(source)
-                for source in custom_sources
-                if source["type"] != "appStoreConnect"
```

Comment:
lets keep this, and the above `filter` for some indefinite time, because there is for sure still project options floating around which still have these config options in them.

Reply:
I'll keep this as it is, and just pass `filter_appconnect=True` to `parse_sources` function on line 449 or 452, depending on which side of the changes you're looking at